src/views/dashboard/device/Device.tsx

```diff
@@ -1,3 +1,5 @@
+"use client";
```

Comment:
I think we should refactor as much of our components to server as possible, but maybe it's not the case here? 🤔